includes/class-wc-gateway-komoju-single-slug.php

```diff
@@ -174,6 +174,10 @@ public function should_use_inline_fields($slug)
 
     public function payment_fields()
     {
+        if (!this->has_fields) {
```

Comment:
It is missing $this <- dollar sign here. Could you change this?

Reply:
:done: Embarrassing 🙈